docs/sql-ref-syntax-ddl-create-table-hiveformat.md

```diff
@@ -65,6 +68,18 @@ as any order. For example, you can write COMMENT table_comment after TBLPROPERTI
 
     Partitions are created on the table, based on the columns specified.
     
+* **CLUSTERED BY**
+
+    Specifies bucket columns for bucketing table.
+    
+* **SORTED BY**
+
+    Used to sort bucket column, we can combine with `ASC` for ascending order, with `DESC` for descending order.
+    
+* **INTO num_buckets BUCKETS**
+
+    Specifies buckets numbers, which is used in  `CLUSTERED BY` clause.
```

Comment:
nit: redundant spaces found between `in`/`CLUSTER BY`. And, `in the CLUSTERED BY clause`?